src/Microsoft.ML.FastTree/FastTreeStatic.cs

```diff
@@ -105,6 +105,48 @@ public static class FastTreeStatic
             return rec.Output;
         }
 
+        /// <summary>
+        /// FastTree <see cref="RankerContext"/>.
```

Comment:
>FastTree <see cref="RankerContext"/>. [](start = 12, length = 37)

Is this intended to be a placeholder? I'm not certain it's terribly helpful.

Reply:
This line appears to still be here. I like the additional line you've added though.

---
In reply to: [221019258](https://github.com/dotnet/machinelearning/pull/1068#discussion_r221019258) [](ancestors = 221019258)

Reply:
duh.. let me remove that first one. 

---
In reply to: [221379761](https://github.com/dotnet/machinelearning/pull/1068#discussion_r221379761) [](ancestors = 221379761,221019258)